chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -293,6 +293,9 @@ pane.item.related.count.singular = %S related:
 pane.item.related.count.plural  = %S related:
 pane.item.parentItem			= Parent Item:
 
+drag.invalid.title 				= Invalid Item
+drag.publications.onlyTopLevel	= Only top-level Zotero items are allowed in My Publications
```

Comment:
Oh, so top-level notes and attachments can't be dragged to My Publications either, so my changed string doesn't work either. "Notes and attachments cannot be dragged to My Publications independently."?

I don't love that, but we don't have a great term the inverse of those — we call them "regular items" internally, but that's not clear at all.